package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -73,6 +74,20 @@ function node_id()
 	return string.gsub(sysconfig.primary_mac, ':', '')
 end
 
+function find_mesh_devices()
```

Comment:
Please call this `get_mesh_devices()`, as it doesn't search for some specific entry like the `find_*` functions in this file do.